autoshop/styles/contact-us.scss

```diff
@@ -0,0 +1,60 @@
+.contact-form {
+  background: #f1f1f1;
+  border: 5px solid #f1f1f1;
+  border-radius: 10px;
+  position: absolute;
+  top: 25%;
+  left: 40%;
+  transition: transform 2s;
+  transition: opacity 2s;
+  transform: scale(200%);
+  opacity: 0;
+  label {
+    display: block;
+    font-family: "Inter", sans-serif;
+    color: black;
+    text-decoration: none;
+    font-size: 20px;
+  }
+  &:hover {
+    opacity: 100%;
+    transform: scale(1);
+  }
+}
+
+.contact-us {
+  width: 100%;
+  height: 760px;
+}
+
+h1 {
```

Comment:
you add quite general selector in this file

are you aware that h1 an ALL pages now will have such styles for h1?